common/isolationgroup/defaultisolationgroupstate/state.go

```diff
@@ -163,22 +169,45 @@ func (z *defaultIsolationGroupStateHandler) get(ctx context.Context, domain stri
 	return ig, nil
 }
 
+func (z *defaultIsolationGroupStateHandler) createAvailableisolationGroupMetricsScope(domainID string, availableIsolationGroups []string) metrics.Scope {
+	domainName, err := z.domainCache.GetDomainName(domainID)
+	if err != nil {
+		domainName = "unknown-domain"
```

Comment:
DomainTag handles empty string properly, so remove this logic and just use empty string